src/dmd/frontend.d

```diff
@@ -27,24 +27,10 @@ void initDMD()
     import dmd.expression : Expression;
     import dmd.objc : Objc;
     import dmd.builtin : builtin_init;
+    import dmd.compiler : addDefaultVersionIdentifiers;
 
     global._init;
-
-    version(linux)
-        global.params.isLinux = 1;
-    else version(OSX)
-        global.params.isOSX = 1;
-    else version(FreeBSD)
-        global.params.isFreeBSD = 1;
-    else version(Windows)
-        global.params.isWindows = 1;
-    else version(Solaris)
-        global.params.isSolaris = 1;
-    else version(OpenBSD)
-        global.params.isOpenBSD = 1;
-    else
-        static assert(0, "OS not supported yet.");
-
+    addDefaultVersionIdentifiers();
```

Comment:
`printPredefinedVersions()` call should go here.

Reply:
Done